### PR TITLE
Upgrade cli to python3 and remove workarounds

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: intelmq/bin/intelmqcli_create_reports.py [hunk 1]
```diff
@@ -20,15 +20,8 @@
 import time
 import zipfile
 
-import psycopg2
```
Comment (aaronkaplan) [inline intelmq/bin/intelmqcli_create_reports.py:23]: not needed anymore? We do need DB access... did it move to lib/intelmqcli.py ?

File: intelmq/bin/intelmqcli.py [hunk 2]
```diff
@@ -424,9 +396,7 @@ def send(self, taxonomy, contact, query, incident_id, requestor=None):
             ziphandle = zipfile.ZipFile(attachment, mode='w',
                                         compression=zipfile.ZIP_DEFLATED)
             data = csvfile.getvalue()
-            if six.PY2:
-                data = unicode(data, 'utf-8')
-            ziphandle.writestr('events.csv', data.encode('utf-8'))
+            ziphandle.writestr('events.csv', data)
```
Comment (aaronkaplan) [inline intelmq/bin/intelmqcli.py:399]: any possibility that this is not UTF-8 ?

**Review by aaronkaplan — APPROVED** [review]
General feedback: much simpler! Go for it

File: intelmq/bin/intelmqcli_create_reports.py (hunk 1 above)
Comment (ghost) [reply intelmq/bin/intelmqcli_create_reports.py:23]: > not needed anymore?

Not in this file. It was only needed to set the extensions which is not necessary anymore as unicode is default.

File: intelmq/bin/intelmqcli.py (hunk 2 above)
Comment (ghost) [reply intelmq/bin/intelmqcli.py:399]: No, according to docs: http://initd.org/psycopg/docs/usage.html#unicode-handling****

**Pull request merged**